pkg/theme/style.go

```diff
@@ -0,0 +1,57 @@
+package theme
```

Comment:
this is just some refactoring to shuffle the code around. When a file has two regions that change at different rates that suggests we should break the file up: in this case we expect this file not to change much but we expect the original file to change more frequently as we support more user-configurable text styles